src/artemis/devices/fluorescence_detector_motion.py

```diff
@@ -0,0 +1,7 @@
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal
+
+
+class FluorescenceDetector(Device):
+
+    pos: EpicsSignal = Cpt(EpicsSignal, "-EA-FLU-01:CTRL")
```

Comment:
Nit: Pos implies a position, which I would assume is a motor but I'm guessing this is just an IN/OUT? In which case could you do something like the backlight where you define:
```
class Backlight(Device):
    OUT = 0
    IN = 1

    pos: EpicsSignal = Component(EpicsSignal, "-EA-BL-01:CTRL")
```